assets/checks.py

```diff
@@ -0,0 +1,28 @@
+from cogs.utils.constants import ids
+
+
+async def perms(perms, g, c, m):
```

Comment:
What are `g`, `c` and `m`, you couldn't have chosen more undescriptive names.
This is unacceptable, these must be changed.

Same goes for lines: 15 and 21